biogtr/io/association_matrix.py

```diff
@@ -22,6 +22,16 @@ class AssociationMatrix:
     ref_instances: list[Instance] = attrs.field()
     query_instances: list[Instance] = attrs.field()
 
+    AVAILABLE_REDUCTIONS = attrs.field(
+        init=False,
+        default={
+            "instance": ["inst", "instance"],
+            "track": ["track", "traj", "trajectory"],
+            None: ["", None],
+        },
+    )
+    AVAILABLE_GROUPINGS = attrs.field(init=False, default=["pred", "gt", None])
```

Comment:
move these to module level and remove attrs since these are just static constants